### PR TITLE
Handle yson32 wire format for decimals inside complex types

### PR DESCRIPTION
**Pull request by chegoryu**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

It is already handled for columns here:

https://github.com/ytsaurus/ytsaurus/blob/4cf39d5b0e61d4909898c1480fa691a93df01138/yt/yt/library/formats/skiff_parser.cpp#L299-L300

So I want to handle this format in complex types too.

### CODE REVIEW

File: yt/yt/tests/integration/formats/test_skiff_format.py [hunk 1]
```diff
@@ -823,8 +853,8 @@ def test_skiff_decimal_inside_composite(self, precision, binary_size, skiff_type
             )
         )
         skiff_data = (
-            b"\x00\x00\x3a\x01" + b"\x00" * (binary_size - 2) + b"\x02\x00\x00\x00" b"pi"
-            + b"\x00\x00\xf1\xfe" + b"\xff" * (binary_size - 2) + b"\x07\x00\x00\x00" b"minus e"
+            self._encode_row(self._get_encoded_decimal_pi(binary_size, skiff_type) + self._encode_string32(b"pi"))
```
Comment (chegoryu) [inline yt/yt/tests/integration/formats/test_skiff_format.py:856]: Without new code in skiff yson converter this error occurs here:

```bash
E   yt.common.YtResponseError: Received response with error
E       Cannot create Skiff parser for table #0
E           Cannot create Skiff parser for column "column"
E               Skiff type yson32 cannot represent type Decimal<35, ?>
```

Comment (dim-an) [reply yt/yt/tests/integration/formats/test_skiff_format.py:856]: Overall looks good to me, but I would like to make a few improvements in tests internally.

Comment (dim-an) [reply yt/yt/tests/integration/formats/test_skiff_format.py:856]: Shipped!

**Comment by dim-an** [comment]
Shipped!

**Comment by dim-an** [comment]
Overall looks good to me, but I would like to make a few improvements in tests internally.

**Comment by dim-an** [comment]
Patch was accepted and merged with minor modifications in test. Thank you!

https://github.com/ytsaurus/ytsaurus/commit/827ef08130edb564c93ac4b63ed663c670ca251f

**Pull request closed without merging**